### PR TITLE
Add real ui and /health

### PR DESCRIPTION
**Pull request by Thib17**
Package static files and templates using github.com/jteeuwen/go-bindata

### CODE REVIEW

File: ui/templates/_base.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse">
+        <ul class="nav navbar-nav mr-auto">
+            <li class="nav-item"><a class="nav-link" href="/">Home</a></li>
+            <li class="nav-item"><a class="nav-link" href="/threads">Threads</a></li>
+            <li class="nav-item"><a class="nav-link" href="/metrics">Metrics</a></li>
+            <li class="nav-item"><a class="nav-link" href="/health">Health</a></li>
```
Comment (iksaif) [inline ui/templates/_base.html]: prometheus uses /-/healthy (and probably /-/threads)

File: ui/templates/_base.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Prometheus Remote Adapater</title>
+
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
```
Comment (iksaif) [inline ui/templates/_base.html]: is there an easy way to package this with bindata ?

File: ui/templates/_base.html (hunk 1 above)
Comment (Thib17) [reply ui/templates/_base.html]: Done

File: ui/templates/_base.html (hunk 2 above)
Comment (Thib17) [reply ui/templates/_base.html]: Done

**Pull request merged**